.github/workflows/pull_request.yml

```diff
@@ -153,14 +203,24 @@ jobs:
       run: ./.github/scripts/setup_git.sh
     - name: Download backend image from artifacts
       id: downloadbackendimage
-      uses: ishworkh/container-image-artifact-download@v2.1.0
+      uses: ishworkh/container-image-artifact-download@v2.0.0
       with:
-        image: "openfoodfacts-server/backend:dev"
-        download_tmp_dir: ${{ runner.temp }}
-    - name: tests
+        image: "openfoodfacts-server/backend:dev"       
+    # downloadbackendimage task loads the image into docker and keeps the original file.
```

Comment:
This shouldn't be necessary for `ishworkh/container-image-artifact-download@v2.1.0`